src/olympia/lib/settings_base.py

```diff
@@ -477,6 +477,7 @@ def get_db_config(environ_var, atomic_requests=True):
     'olympia.devhub',
     'olympia.discovery',
     'olympia.files',
+    'olympia.git',
```

Comment:
I'm not sure about creating a new top level app for this - its more an interface to git so `olympia.lib.git` seems more appropriate to me... wdyt @diox?

Reply:
As I said, we'll need more code like new tasks and a new admin so I think that would make sense. And I rather see `lib.git` moved to `git.utils`.